app/Models/BankDetails.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class BankDetails extends Model
+{
+    use HasFactory;
+    protected $table = 'bank_details';
+    protected $fillable = ['id', 'key', 'label', 'value'];
```

Comment:
simply use guarded function when you want all field to be fillable
```suggestion
    protected $guarded = [];
```